### DISCUSSION TITLE
How to create multiple jobs in Quartz dynamically in C#?

### DISCUSSION THREAD

**Discussion by nirmalk323** [quartznet/quartznet#965] [General]
I am new to Quartz.net. I have a requirement that I need to schedule multiple reports through UI by the client as per their convenient timing.

We have almost 30 clients, and each client has 20-25 reports. Clients have access to the portal and they can configure their report timing by any day/monthly/yearly.

For example, if client A scheduled 5 reports:

report a ---> daily @ 10PM
report b ---> monthly once on date 02 @ 10AM
report c ---> yearly once on date 30/june @ 5PM
report d ---> daily @ 12PM
report e ---> monthly once on date 02 @ 10AM
In the same way, all clients have the privilege to configure their reports respectively from the UI.

Could you please help me on how to create multiple jobs dynamically from UI using Quartz.net?

**Comment by lahma** [comment]
You should probably have a look at either [Quartzmin](https://github.com/jlucansky/Quartzmin) or [Crystal Quartz](https://github.com/guryanovev/CrystalQuartz), they are UI's doing this exact thing.

**Reply by nirmalk323** [reply]
Hi @lahma , Thanks for the information. I will check Quartzmin and crystal quartz. 
In my case, I have created a new UI to get the information related to the report scheduling from the client and storing the same in our database.

Now, I need to use that information and send the report as per the timing scheduled/stored in database

Do we can still use quartzmin for the above approach?
